config/servicemappings/alloydb.yaml

```diff
@@ -142,6 +143,16 @@ spec:
             version: v1beta1
             kind: AlloyDBCluster
           targetField: name
+        - tfField: secondary_config.primary_cluster_name
+          description: |-
+            "(Required) Name of the primary cluster must be in the format
```

Comment:
Nit: `(Required)` is probably not necessary because whether a field is required or optional is represented by the required list in the schema.

Reply:
Removed `(Required)`